plugins/im-export-breakpoints/_include-media-export-breakpoints.scss

```diff
@@ -0,0 +1,24 @@
+@function im-breakpoints-to-json($target-name) {
+  $json: '';
+  $target-value: map-get($breakpoints, $target-name);
+
+  @each $name, $value in $breakpoints {
+    $json: if($json == '', '{', $json + ',');
+    $state: $target-value >= $value;
+    $json: $json + '"#{$name}":#{$state}';
+  }
+
+  $json: $json + '}';
+
+  @return $json;
+}
+
+@mixin im-export-breakpoints($element: body) {
+  @each $name, $value in $breakpoints {
+    @include media('>=#{$name}') {
+      #{$element} {
```

Comment:
I think in general `<body>` is a good option because it's faster to query. Also, [`support for getComputedStyle()`] on pseudo-elements is slightly more restrictive than on normal elements. But it's worth thinking about.